### PR TITLE
Added support of list-all-packages

### PR DESCRIPTION
**Pull request by rahul2393**
[Issue](https://github.com/aquasecurity/trivy/issues/548)

Screenshot 2020-07-28 at 8 51 13 PM

### CODE REVIEW

File: pkg/report/writer.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 
 	"golang.org/x/xerrors"
 
+	fanalTypes "github.com/aquasecurity/fanal/types"
```
Comment (knqyf263) [inline pkg/report/writer.go:16]: nit: in other packages, we use `ftypes`.

Comment (rahul2393) [reply pkg/report/writer.go]: @knqyf263 Done

File: internal/app.go [hunk 2]
```diff
@@ -182,6 +182,12 @@ var (
 		EnvVars: []string{"TRIVY_IGNORE_POLICY"},
 	}
 
+	listAllPackages = cli.BoolFlag{
+		Name:    "list-all-pkgs",
+		Usage:   "enabling the option will output all packages regardless of vulnerability",
+		EnvVars: []string{"TRIVY_LIST_ALL_PACKAGES"},
```
Comment (aytekinar) [inline internal/app.go:188]:
```suggestion
		EnvVars: []string{"TRIVY_LIST_ALL_PKGS"},
```
Hi. I am no expert in Golang, and hence, I am not sure if I need to provide a similar suggestion elsewhere in the PR.

I have just noticed this and wanted to give you heads-up for "consistency" reasons. A similar CLI switch is `--removed-pkgs` with the corresponding environment variable `TRIVY_REMOVED_PKGS`. Perhaps, you would like this one to be shorter, too.

**Comment by aytekinar** [comment]
Wow! Thank you for the fast response after seeing my issue ❤️

File: internal/app.go (hunk 2 above)
Comment (rahul2393) [reply internal/app.go]: Changes pushed, thanks for suggestion

**Comment by knqyf263** [comment]
Thanks!

**Pull request merged**